tests/tekton-tasks_test.go

```diff
@@ -242,4 +244,33 @@ var _ = Describe("Tekton-tasks", func() {
 			}, tenSecondTimeout, time.Second).Should(BeTrue(), "there should be no role bindings left")
 		})
 	})
+	Context("multiple CRs deployed", func() {
+		AfterEach(func() {
+			// To reset TTO we need to delete it and deploy it again
+			tto := strategy.GetTTO()
+			apiClient.Delete(ctx, tto)
+			strategy.CreateTTOIfNeeded()
+			waitUntilDeployed()
+		})
+
+		It("check if correct status is set for TTO", func() {
+			tto1 := strategy.createTekton("tto-test-1")
+			tto2 := strategy.createTekton("tto-test-2")
+
+			tektonTasksCRList := &tekton.TektonTasksList{}
+			tektonTasksCRList.SetGroupVersionKind(schema.GroupVersionKind{
+				Kind:    "TektonTasks",
+				Version: "tektontasks.kubevirt.io/v1alpha1",
+			})
+			apiClient.List(ctx, tektonTasksCRList)
+
+			for _, item := range tektonTasksCRList.Items {
+				Expect(item.Status.Conditions[0].Status).To(Equal(v1.ConditionFalse))
```

Comment:
#Please change the check for `Conditions` to be based on the `Type` of the condition instead of slice indexes. You can also check for the exact `Reason` of the condition.